src/corefx/System.Globalization.Native/errors.h

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+#pragma once
+
+#include <unicode/utypes.h>
+
+/*
+* These values should be kept in sync with
+* Interop.GlobalizationInterop.ResultCode
+*/
+enum ResultCode : int32_t
+{
+    Success = 0,
+    UnknownError = 1,
+    InsufficentBuffer = 2,
+};
+
+/*
+Function:
+GetCalendarDataResult
```

Comment:
I need to update this name.  Maybe just remove this section all together. It is redundant with the method declaration.